Src/Support/Google.Apis.Tests/Apis/Requests/ClientServiceRequestTest.cs

```diff
@@ -1197,6 +1197,23 @@ private class TestHttpHandler : HttpMessageHandler
         public static TheoryData<HttpContent, HttpStatusCode, string, string> DeserializeErrorThrows =>
             new TheoryData<HttpContent, HttpStatusCode, string, string>
             {
+                {
+                    new StringContent(
+                        "{ " +
+                        "error:\"invalid_target\", " +
```

Comment:
It would be good to put `error` and `error_description` into quotes so that it's fully valid JSON.

Reply:
(And possibly extract out the literal so you can use it again in the assertion?)

Reply:
Done, we'll wait on green and push.